core/src/main/java/org/mifos/mobile/core/ui/component/CopyrightItemContent.kt

```diff
@@ -0,0 +1,28 @@
+package org.mifos.mobile.core.ui.component
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun CopyrightItemContent(name: Int, copyrightText: String) {
```

Comment:
You don't need to create this Composable, the `AboutUsItemCard` should handle this cases where there is a title and `subtitle`